client/v2/autocli/common.go

```diff
@@ -58,9 +61,13 @@ func (b *Builder) buildMethodCommandCommon(descriptor protoreflect.MethodDescrip
 		return exec(cmd, input)
 	}
 
-	if b.AddQueryConnFlags != nil {
-		b.AddQueryConnFlags(cmd)
-	}
+	//if b.AddQueryConnFlags != nil {
```

Comment:
why is this commented?

Reply:
good catch I moved it to query because if I include TX there I will get a clash of redefinition errors for some flags.